controllers/connectionpool_controller.go

```diff
@@ -170,16 +177,17 @@ func (h ConnectionPoolHandler) get(ctx context.Context, avn *aiven.Client, obj c
 
 	prefix := getSecretPrefix(connPool)
 	stringData := map[string]string{
+		prefix + "NAME":         connPool.Name,
 		prefix + "HOST":         s.URIParams["host"],
-		prefix + "PORT":         s.URIParams["port"],
```

Comment:
I have only one question - what was the problem passing PORT as it arrives from the `URIParams["port"]`? Now we pars the URI `url.Parse(cp.ConnectionURI)` and get it from there

Reply:
> I have only one question - what was the problem passing PORT as it arrives from the `URIParams["port"]`? Now we pars the URI `url.Parse(cp.ConnectionURI)` and get it from there

Well. Service has its own port. And the pool has its own one. You create a pool, you want to use its port. Currently you create a pool and still use the one that comes with the service. So not really using the pool.

Reply:
I'll add a comment there.

Reply:
Got it thanks for the clarification